scripts/superdesk-monitoring/monitoring.js

```diff
@@ -122,13 +122,13 @@
                             {term: {'task.last_authoring_desk': desk_id}},
                             {and: [
                                 {term: {'task.desk': desk_id}},
-                                {terms: {state: ['scheduled', 'published', 'corrected', 'killed']}}
```

Comment:
I think that in order to don't confuse the users I will make scheduled stage configurable. By default it is disabled(not visible at all in the app) and out stage show also scheduled items, but if it is enabled the scheduled items will not be showed only on scheduled stage.